streaming/src/main/scala/org/apache/spark/streaming/scheduler/ReceivedBlockTracker.scala

```diff
@@ -40,7 +42,8 @@ private[streaming] case class BatchAllocationEvent(time: Time, allocatedBlocks:
   extends ReceivedBlockTrackerLogEvent
 private[streaming] case class BatchCleanupEvent(times: Seq[Time])
   extends ReceivedBlockTrackerLogEvent
-
+private[streaming] case class CombinedRBTLogEvent(events: List[ReceivedBlockTrackerLogEvent])
```

Comment:
nit: I prefer to use `CombinedReceivedBlockTrackerLogEvent` since we don't use `RBT` in other places and the meaning of `RBT` is not obvious.